addons/l10n_ar/views/report_invoice.xml

```diff
@@ -241,6 +241,13 @@
             </div>
         </xpath>
 
+        <!-- Show total amount in letters for MiPyMEs document types -->
+        <xpath expr="//div[@id='total']/div/table" position="after">
+            <t t-if="o.l10n_latam_document_type_id.code in ['201', '202', '203', '206', '207', '208', '211', '212', '213']">
+                <strong>Son: </strong><span t-esc="o.currency_id.with_context(lang=o.partner_id.lang).amount_to_text(o.amount_total)"/>
```

Comment:
@Bruno-Zanotti This is applicable in all languages?  (or should we always do Spanish?)

Reply:
- Is a DOC only used in AR at it will be probably always in SPA.

Reply:
Are we talking about this document? https://www.cnv.gov.ar/descargas/web/blob/06ADF721-130F-4AF4-B607-2AD51E1CA23A

I didn't find anything about the amount in letters in it, can you point the right sections?
I also think that if the Argentinean government is asking for something like that, it should always be in Spanish and we should therefore not look at the client's language.

Reply:
@william-andre http://biblioteca.afip.gob.ar/dcp/LEY_C_027440_2018_05_09 "Artiiculo 5" letter f)
@Bruno-Zanotti will modify the language

Reply:
Done! @vbe-odoo @william-andre